docs/integrations-in-rancher/cluster-api/overview.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Overview
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/integrations-in-rancher/cluster-api/overview"/>
+</head>
+
+## Demo
```

Comment:
I decided to move the demo to the UI section as a subsection. It covers how to install, create, and import CAPI clusters, and add monitoring to CAPI clusters all through the UI so I thought it would be a better fit there.